### PR TITLE
Added OpenSSL version requirement to README.

### PR DESCRIPTION
**Pull request by WilliamDenniss**

### CODE REVIEW

**Comment by WilliamDenniss** [comment]
I signed it!

File: README.md [hunk 1]
```diff
@@ -6,6 +6,15 @@
 
 Python client for the [kubernetes](http://kubernetes.io/) API.
 
+## Requirements
+
+A version of python with OpenSSL version 1.0.0 or later.
```
Comment (mbohlool) [inline README.md]: I am not sure if this is the minimum requirement. Worth mentioning this is on mac. What we know is, the client does not work with current OPENSSL_VERSION default on osx.

Comment (WilliamDenniss) [inline README.md]: Ah ok, should I make this a macOS-specific instruction then? If so, I'd mention using brew as well.

Comment (mbohlool) [inline README.md]: Yes, that's more appropriate I think. Thanks.

**Comment by mbohlool** [comment]
@WilliamDenniss Hey William, are you going to update this PR? I can send another one to fix it if you don't have time.

**Pull request merged**